.drone.yml

```
@@ -30,7 +37,7 @@ steps:
         from_secret: QUAY_ROBOT_TOKEN
       DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
     depends_on:
-      - build project
+      - test project
```

I don't think it does actually depend on test. We can speed up the pipeline by doing it in parallel with the tests? Maybe?
```suggestion
      - build project
```

I guess for `:latest` maybe we do want to ensure it's only pushing on a good build? Or perhaps we think having `:latest` be `HEAD` as much as possible, regardless of if its broken, is a better approach?

---

My thinking is we shouldn't be pushing broken builds up at all - if someone does come to use it and it's not passed tests then it's not really usable. Its not like the source code that can be debugged effectively. 

I don't believe the speed benefits are worth the fact that we risk having broken build images uploaded.

---

I agree, we should not be pushing build that have failed tests. They are faulty and increase the risk of being deployed to an environment later.

---

Yeah, I guess ideally we'd have "build Docker image" and "push Docker image" as two things with different dependencies, but we can't do that (simply) here.